bodhi/server/models.py

```diff
@@ -1248,6 +1305,43 @@ class Build(Base):
         'polymorphic_identity': ContentType.base,
     }
 
+    # Local cache in case we need more koji info
+    _kojiinfo = None
+
+    def _get_kojiinfo(self):
+        if not self._kojiinfo:
+            koji_session = buildsys.get_session()
+            self._kojiinfo = koji_session.getBuild(self.nvr)
+        return self._kojiinfo
+
+    def _get_n_v_r(self):
+        """Return the N, V and R components of an RPM build from NVR."""
+        return self.nvr.rsplit('-', 2)
+
+    @property
```

Comment:
Can't you use `self._kojiinfo['name']`, `self._kojiinfo['version']`, `self._kojiinfo['release']` also here? See my comment for `ModuleBuild`.

Reply:
Using the `_get_kojiinfo` function requires going out to koji, which for RPMs (and containers) is not required, since the `rsplit` method works just fine there.

Reply:
Ah, that makes sense, I thought you are calling that at least once for these builds anyway. +1 then.